src/lib/final_report.ml

```diff
@@ -417,6 +443,11 @@ module To_workflow
             let wf = get_vcf repr in
             let title = sprintf "VCF: %s" name in
             title, Some wf#product#path)
+        @ (Option.value_map ~default:(["Kallisto", None]) kallisto ~f:(fun ks ->
```

Comment:
Ah you need the string Kallisto in there somehow; then it could be something like

["Kallisto"; ["title", path; ...]] instead (that way you're not overloading what items in your list mean, and don't need an option type in the list.)